lib/LangInterface.php

```diff
@@ -468,6 +530,7 @@ public static function change_post_language( int $post_id, string $lang ) : bool
 		$meta_keys_to_translate = apply_filters( 'ubb_change_language_post_meta_translate_keys', $default_meta, $post_id, $lang, $old_lang );
 		if ( ! self::translate_post_meta( $post_id, $lang, $meta_keys_to_translate ) ) {
 			// TODO: Failure state
+			// FIXME: language is changed but meta is not, what to do and return?
```

Comment:
Revert and return an error seems to be the best option.

Reply:
Added issue #99 for this to be done at a later time.